### ISSUE TITLE
On the exhaustiveness of branches & polarity

### ISSUE THREAD

**Issue by joshnguyen99** [kmjohnson/twitter-morals#3]
Hi Kristen,

I have two questions regarding the moral foundation branches and polarity.

First, for tweets that are identified with 2 branches, shouldn't they contain two values for polarity as well? Or does the polarity only correspond to the first branch? 

My current understanding is if a tweet's label is `(branch1, branch2, polarity) = (2, 3, 1)`, this translates to `fairness (branch=2, polarity=1)` and `loyalty (branch=3, polarity=1)`. Could you confirm whether this understanding is correct?

Second, the paper states that a "secondary foundation" is allowed to be annotated for a tweet. Does it mean that if a tweet contains two branches, those are the only labels it should have, according to the annotators?

I ask the second question because I'm interested in the absence of moral foundations in a tweet as well. However, the restriction to a maximum of 2 possible branches does not seem to guarantee that branches 3, 4, 5 aren't at all possible.

Your response is much appreciated!

**Comment by kmjohnson** [comment]
Hello, I apologize for missing your question when you first posted it. 

Your understanding is correct. When we were labeling the tweets, we initially had a column for polarity2 to go with branch2. However, the polarities always seemed to align. There may be a handful of tweets that will have two polarities listed, because I do not remember if we removed those from the training data or not. 

For the second question, not necessarily. The foundations can be hard to classify, especially in political tweets. There may be a third foundation, but for simplicity we chose only the main two. Typically we didn't see a large amount of more-than-two foundations present in the tweets. 

I think it's possible to have more than 2 foundations, as well as tweets that express thoughts without a moral frame, especially in longer tweets. Tweets can often be compound in nature, that's why we used 2 frames in the previous paper and then 2 foundations in this paper.
